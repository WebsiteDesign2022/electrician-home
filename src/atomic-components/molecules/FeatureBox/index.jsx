import React from 'react';

import './index.scss';

const FeatureBox = ({ title, description }) => (
  <div className="col d-flex align-items-start">
      <div className="icon-square flex-shrink-0 me-3 py-1 arrow">
        {">>"}
      </div>
      <div className="align-items-start">
        <h2 align="left" className="title">{title}</h2>
        <p align="left" className="description">{description}</p>
      </div>
  </div>
);

export default FeatureBox;