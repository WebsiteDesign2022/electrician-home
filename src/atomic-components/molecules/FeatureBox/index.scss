.arrow {
    color: orange;
}

.title {
    color: white;
}

.description {
    color: white;
}
