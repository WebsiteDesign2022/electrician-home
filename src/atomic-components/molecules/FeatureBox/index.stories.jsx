import React from 'react';

import FeatureBox from '.';

export default {
  title: 'Electrician-Home/molecules/FeatureBox',
  component: FeatureBox,
  parameters: {
    // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
    layout: 'fullscreen',
  },
};

const Template = (args) => <FeatureBox {...args} />;

export const Primary = Template.bind({});
// More on args: https://storybook.js.org/docs/react/writing-stories/args
Primary.args = {

};
