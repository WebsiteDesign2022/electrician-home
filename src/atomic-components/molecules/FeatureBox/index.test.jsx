import { render } from '@testing-library/react'

import FeatureBox from '.';

describe("FeatureBox", () => {
    test("renders FeatureBox", () => {
        render(
            <FeatureBox
                title="FeatureBox-title"
                description="FeatureBox-description"
            />
        );

        expect(screen).toMatchSnapshot();

    });
});
