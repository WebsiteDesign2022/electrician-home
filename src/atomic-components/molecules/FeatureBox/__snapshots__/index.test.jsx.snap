// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatureBox renders FeatureBox 1`] = `Screen {}`;
